python/lsst/meas/astrom/directMatch.py

```diff
@@ -31,8 +33,8 @@ class DirectMatchTask(Task):
 
     Parameters
     ----------
-    butler : `lsst.daf.persistence.Butler`
-        Data butler containing the relevant reference catalog data.
+    butler : `None`
+        Compatibility parameter. Should not be used.
     refObjLoader : `lsst.meas.algorithms.LoadReferenceObjectsTask` or `None`
```

Comment:
This should be `lsst.meas.algorithms.ReferenceObjectLoader`, and below in `setRefObjLoader` that should only take `lsst.meas.algorithms.ReferenceObjectLoader` and the docstring can be simplified.

Reply:
I think I've cleaned that up now.